compiler/rustc_mir_build/src/build/expr/as_operand.rs

```diff
@@ -42,15 +42,16 @@ impl<'a, 'tcx> Builder<'a, 'tcx> {
     /// We tweak the handling of parameters of unsized type slightly to avoid the need to create a
     /// local variable of unsized type. For example, consider this program:
     ///
-    /// ```rust
-    /// fn foo(p: dyn Debug) { ... }
+    /// ```ignore HELP (should that p in bar be box_p?)
```

Comment:
yes. but it still fails to compile after - maybe the original code meant to use `foo(&*box_p)` instead? `dyn Debug` can't be passed to a function.

(either that, or this is left over from some pre-1.0 code.)

Reply:
I added `#![feature(unsized_locals, unsized_fn_params)]` and it worked. Does that seem like a bad fix?